dashboard/app/controllers/sections_controller.rb

```diff
@@ -11,8 +11,6 @@ def new
   def edit
     return head :forbidden unless current_user&.admin
 
-    redirect_to '/home' unless params[:loginType] && params[:participantType]
```

Comment:
This will set us up for the flow *into* this edit page, where we don't need to add in the params to the url